app/views/slides/_thumbnail_template.html.erb

```diff
@@ -19,7 +19,7 @@
             <div class="slidecard-description"><%= strip_tags(slide.description).truncate(80) %></div>
             </div>
             <div class="slidecard-information">
-              <div class="h5" style="margin-top:2px;margin-bottom:2px;">by <a href="<%= user_path(slide.user_id) %>"><%= strip_tags slide.user_display_name %></a></div>
+              <div class="h5" style="margin-top:2px;margin-bottom:2px;">by <a href="/<%= slide.user.username %>"><%= strip_tags slide.user_display_name %></a></div>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)